### PR TITLE
Refactor scripts for ease of unittesting the db dump recent script 

### PR DESCRIPTION
**Pull request by gregcorbett**
~~Depends on #23~~

- Ensures all timestamps are treated as UTC
- Moves existing functionality of check_db_dump_recent.py into a class for ease of unit testings
- Updates the README.md

### CODE REVIEW

File: check/check_db_dump_recent.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+
+"""
+This file contains a script to check the GOCDB failover process is happening.
+
+Specifically, it checks the log file passed as the first arguement for evidence
```
Comment (Sae126V) [inline check/check_db_dump_recent.py:6]: Seems there is a typo for arguement -> argument and suceeded -> succeeded.

File: importDBdmpFile/1_runDbUpdate.sh [hunk 2]
```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
 
-# Script for downloading the gocdb5 db dmp file and importing into the 
-# local oracle xe instance using sqlplus. 
-# Notes: 
-# - The updatelog.txt file is appended each time with log messages whenever the script is ran. 
+# Script for downloading the gocdb5 db dmp file and importing into the
```
Comment (Sae126V) [inline importDBdmpFile/1_runDbUpdate.sh:3]: I see that there is a PR #23  addressing whitespaces. Is it being repeated here ?

File: check/check_db_dump_recent.py (hunk 1 above)
Comment (gregcorbett) [reply check/check_db_dump_recent.py]: Fixed in [db4a211](https://github.com/GOCDB/gocdb-failover-scripts/pull/22/commits/db4a211a3c4c5ee479bf47236b305dcb85c342af)

**Comment by Sae126V** [comment]
It seems there is an empty file: check/__init__.py. Is it intentional ?

**Comment by Sae126V** [comment]
I finished my review. LGTM

**Comment by gregcorbett** [comment]
> It seems there is an empty file: check/**init**.py. Is it intentional ?

Yes, it is used to mark a directory as a Python package so that unit tests will be able to import it.

File: check/check_db_dump_recent.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                last_success = datetime.strptime(
+                    last_line,
+                    "%Y-%m-%dT%H:%M:%S%z",
+                )
+            except ValueError:
+                print("An error occured: {0}".format(last_line))
+                return RETURN_CODE_CRITICAL
+
+            print("The failover process last suceeded at %s" % last_success)
+            # If the failover process hasn't suceeded in a while, the
```
[the remark this answers is not in the document]
Comment (gregcorbett) [reply check/check_db_dump_recent.py]: fixed in [e0b8f84](https://github.com/GOCDB/gocdb-failover-scripts/pull/22/commits/e0b8f84d7314bea38ae801e0de4ea9664d489e51)

**Comment by gregcorbett** [comment]
I have rebased on top of the latest `main` (which has removed the whitespace commit already applied in #21) and squashed the SPG fixes into one commit.

**Review by Sae126V — APPROVED** [review]

**Pull request merged**